src/objectives/elbo/entropy.jl

```diff
@@ -0,0 +1,34 @@
+
+struct ClosedFormEntropy <: AbstractEntropyEstimator end
+
+function (::ClosedFormEntropy)(q, ::AbstractMatrix)
```

Comment:
Same here: if we add an `estimate_entropy` method which takes a `AbstractEntropyEstimator` as the first argument, then we can just document `estimate_entropy` and list the entropy estimators, making discoverability quite easy for the user?